frontend/src/components/Dashboard.vue

```diff
@@ -0,0 +1,321 @@
+                        permissions levels
+                    </span>
+                </div>
+
+                <span class="p-input-icon-right">
+                    <i class="pi pi-search" />
+                    <InputText
+                        class="dash-search"
+                        v-model="filters['global'].value"
+                    />
+                </span>
+
+                <DataTable
+                    v-model:filters="filters"
+                    :value="userRoleAssignments"
+                    paginator
+                    :rows="5"
+                    :rowsPerPageOptions="[5, 10, 15, 20, 50, 100]"
+                    dataKey="id"
+                    filterDisplay="menu"
+                    :loading="loading"
+                    :globalFilterFields="[
+                        'user.user_name',
+                        'role.parent_role.role_name',
+                        'role.role_name',
+                        'role.client_number.forest_client_number',
+                    ]"
+                    paginatorTemplate="RowsPerPageDropdown CurrentPageReport PrevPageLink NextPageLink"
+                    currentPageReportTemplate="{currentPage} of {totalPages} pages"
+                >
+                    <template #empty> No application selected. </template>
+                    <template #loading>
+                        Loading customers data. Please wait.
+                    </template>
+                    <Column header="User name" sortable field="user.user_name">
+                        <template #body="{ data }">
+                            <Icon icon="AvatarFilledIcon" medium />
+                            <span class="span-icon">
+                                {{ data.user.user_name }}
+                            </span>
+                        </template>
+                    </Column>
+                    <Column
+                        field="user.user_type.description"
+                        header="Domain"
+                        sortable
+                    ></Column>
+                    <Column field="role.role_name" header="Role" sortable>
+                        <template #body="{ data }">
+                            {{
+                                data.role.parent_role
+                                    ? data.role.parent_role.role_name
+                                    : data.role.role_name
+                            }}
+                        </template></Column
+                    >
+                    <Column
+                        field="role.client_number.forest_client_number"
+                        header="Forest Client ID"
+                        sortable
+                    ></Column>
+                    <Column>
+                        <template #body="{ data }">
+                            <button
+                                class="btn btn-icon"
+                                @click="tryDelete(data)"
```

Comment:
Since this button is intended for deletion, I prefer not to prefix it with "try".
Maybe give it some context for the name then just "delete"; like "deleteUserAssignment" or something similiar.

Reply:
This name is coming from what we had from ManageAccess.vue. You are right about the naming and I'll be changing it. Thanks.